dnfdaemon-client/context.hpp

```diff
@@ -88,10 +88,10 @@ class Context {
     std::unique_ptr<sdbus::IProxy> session_proxy;
 
     // global command line arguments
-    std::unique_ptr<libdnf::OptionBool> verbose = std::make_unique<libdnf::OptionBool>(false);
-    std::unique_ptr<libdnf::OptionBool> assume_yes = std::make_unique<libdnf::OptionBool>(false);
-    std::unique_ptr<libdnf::OptionBool> assume_no = std::make_unique<libdnf::OptionBool>(false);
-    std::unique_ptr<libdnf::OptionBool> allow_erasing = std::make_unique<libdnf::OptionBool>(false);
+    libdnf::OptionBool verbose {false};
+    libdnf::OptionBool assume_yes {false};
+    libdnf::OptionBool assume_no {false};
+    libdnf::OptionBool allow_erasing {false};
```

Comment:
I believe that there are extra spaces `libdnf::OptionBool allow_erasing{false};`? To ensure what is correct I recommend to use `clang-format -i <path to file>`.